.changeset/orange-yaks-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Fixed a bug in the Popover component by applying z-index to the Popper element.
```

Comment:
Changing the `z-index` is an implementation detail as is the fact that the popover component is using Popper under the hood. Always ask yourself: what information is relevant to the users of Circuit UI?

```suggestion
Fixed a bug in the Popover component to correctly stack the popover content on top of the overlay on mobile.
```

Reply:
Thank you 🌟. Will update it!